packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -685,6 +685,12 @@ Future<T> showMenu<T>({
 /// Used by [PopupMenuButton.onSelected].
 typedef void PopupMenuItemSelected<T>(T value);
 
+/// Signature for the callback invoked when a [PopupMenuButton] is dismissed
+/// without selecting an item.
+///
+/// Used by [PopupMenuButton.onCancelled].
+typedef void PopupMenuCancelled<T>();
```

Comment:
no need to declare `T` here, since you don't use it

Reply:
Removed